pkg/ddc/jindofsx/transform.go

```diff
@@ -365,6 +365,17 @@ func (e *JindoFSxEngine) transformResources(runtime *datav1alpha1.JindoRuntime,
 		}
 	}
 
+	// mem set request
+	if e.getTieredStoreType(runtime) == 0 {
+		quotaString := strings.TrimRight(userQuotas, "g")
+		if quotaString != "" {
+			i, _ := strconv.Atoi(quotaString)
+			if i > defaultMemLimit {
```

Comment:
Just wonder what is the unit for `defaultMemLimit`? Is it a GiB unit? So that means, when user setting a >100Gi quota, we set Master memory request to 30Gi?

Reply:
First of all, when many customers use the MEM mode of JindoFSx, they will set the path of the TierStore to /dev/shm, then the memory usage of the ns service needs to be guaranteed. When the quota>100G, we will assume that the user is a massive file. Scenario, the number of files may be between one million and ten million, then the usage of ns will be about 30G at this time. This is an empirical value. If the ns service is not enough, eviction and crash will occur